models/template_helpers_test.go

```diff
@@ -137,6 +137,27 @@ func TestRuleTemplates(t *testing.T) {
 		assert.Equal(t, cParams.Source, updated.Source)
 	})
 
+	t.Run("change err - mismatch names", func(t *testing.T) {
+		tx, err := db.Begin()
+		require.NoError(t, err)
+		defer func() {
+			require.NoError(t, tx.Rollback())
+		}()
+
+		q := tx.Querier
+
+		name := gofakeit.UUID()
+
+		cParams := createTemplateParams(name)
+		_, err = models.CreateTemplate(q, cParams)
+		require.NoError(t, err)
+
+		uParams := changeTemplateParams(name)
```

Comment:
cParams - Isn't this style of variable name consistency breach? The variable is name is not self-explaining, a comment could clarify that. BTW - 'u' at the beginning usually stands for unsigned.

Reply:
It's test. And everywhere in test files is used as this.
I don't get why comment is needed for name of variable, it's in little scope.

Variable prefix based on types I would say are deprecated. You got IDE for it.

Reply:
IMHO Hungarian notation is antipattern it 2021. Here `uPrams' stands for `update params`, as for me it's short and descriptive enough.

Reply:
I used pExporter for postgres exporter and I was told to change it.
As I said I don't have problems with that. But at the same time it could be updParams or updateParams to be clearer.